lms/djangoapps/discussion_api/tests/test_api.py

```diff
@@ -1427,6 +1427,45 @@ class CreateThreadTest(
         SharedModuleStoreTestCase,
         MockSignalHandlerMixin
 ):
+    LONG_TITLE = (
```

Comment:
Justification for hard coding in a long title instead of just generating one with something like `LONG_TITLE = 'x' * 2000`:

I want to be sure that we test that the string isn't just being truncated the correct *length*, but it's being truncated to the correct *string* as well. What I mean by that is if I had erroneously written [this](https://github.com/edx/edx-platform/pull/15237/files#diff-1f8b9b8344cecfd83d74693377466fd4R100) line as `event_data['title'] = full_title[-TRACKING_MAX_FORUM_TITLE:]`, doing this test with `'xxxxx...'` would pass, whereas with the lorem ipsum text it would fail.

Additionally, I didn't want to randomly generate a string, because:
 * Tests should be deterministic
 * It's easier to debug this test if the text is spelled out explicitly